### PR TITLE
chore: rename header guards (#945)

### PR DESCRIPTION
**Pull request by zrlk**
We need a stronger tag than `chore` for changes like this

### CODE REVIEW

**Comment by danielmoy-google** [comment]
> We need a stronger tag than `chore` for changes like this

`migration:` or `refactor:` or something

File: kythe/cxx/extractor/language.h [hunk 1]
```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#ifndef KYTHE_CXX_COMMON_KYTHE_LANGUAGE_H_
-#define KYTHE_CXX_COMMON_KYTHE_LANGUAGE_H_
+#ifndef KYTHE_CXX_EXTRACTOR_KYTHE_LANGUAGE_H_
```
Comment (salguarnieri) [inline kythe/cxx/extractor/language.h]: No second KYTHE

Comment (zrlk) [reply kythe/cxx/extractor/language.h]: I'd prefer `toil:`

**Comment by schroederc** [comment]
`refactor` is an acceptable type.

**Pull request merged**